src/index.js

```diff
@@ -32,7 +32,7 @@ export { default as Icon } from './Icon';
 export { default as IconButton } from './IconButton';
 export { default as Input, InputLabel, InputAdornment } from './Input';
 export { default as Grid } from './Grid';
-export { GridList, GridListTile, GridListTileBar } from './GridList';
+export { default as GridList, GridListTile, GridListTileBar } from './GridList';
```

Comment:
I agree this is actually nicer.

Perhaps named `GridList` export from `GridList/index.*` should also be removed.

Reply:
Yes, we should for consistency. Do you want to fix it? :)